docs/filter-functions.md

```diff
@@ -237,6 +237,30 @@ For example, check if all changes except for `config.json` files are formatting:
 ```yaml+jinja
 {{ source.diff.files | reject(attr='new_file', regex=r/config\.json$/) | isFormattingChange }}
 ```
+#### `intersection`
+
```

Comment:
should be after `includes` before `map` 

// I am trying the new rule to see if i get the label, so being over verbose here...